docs/datasets.rst

```diff
@@ -14,6 +14,8 @@ datasets. Currently available datasets are:
 - :func:`clouddrift.datasets.gdp6h`: 6-hourly GDP data from a ragged-array
   NetCDF file hosted by the public HTTPS server at
   `NOAA's Atlantic Oceanographic and Meteorological Laboratory (AOML) <https://www.aoml.noaa.gov/phod/gdp/index.php>`_.
+- :func:`clouddrift.datasets.gdp_source`: source GDP data without being pre-processed unlike
```

Comment:
Change to:
Source GDP data from which the 6-hourly and 1-hourly products are derived.